dev/README_RELEASE_PROVIDER_PACKAGES.md

```diff
@@ -1197,6 +1236,38 @@ do
  svn mv "${file}" "${base_file//rc[0-9]/}"
 done
 
+# Delete the old `pypi-rc` providers
+svn rm ${SOURCE_DIR}/pypi-rc/*
```

Comment:
svn commit -m "Promote RC artifacts for Airflow Providers to released $(date "+%Y-%m-%d%n")"